03_launch_mgmt_cluster.sh

```diff
@@ -18,6 +19,9 @@ CAPBMREPO="${CAPBMREPO:-https://github.com/metal3-io/cluster-api-provider-bareme
 CAPBMBRANCH="${CAPBMBRANCH:-master}"
 FORCE_REPO_UPDATE="${FORCE_REPO_UPDATE:-false}"
 
+BMO_RUN_LOCAL="${BMO_RUN_LOCAL:-false}"
+CAPBM_RUN_LOCAL="${CAPBM_RUN_LOCAL:-false}"
```

Comment:
Can you add docs for these new variables to `config_user.sh` ?

Reply:
done. I also added a test to check that they are running locally